ubuntu-core-20-amd64-intel-iotg-beta.json

```diff
@@ -0,0 +1,38 @@
+{
+    "type": "model",
+    "series": "16",
+    "authority-id": "canonical",
+    "brand-id": "canonical",
+    "revision": "1",
```

Comment:
revision 0 is implied, revision 1 is only to be used when we need to supersede old assertion with a new one. If this is the initial model, it's best to have this as "0" or omitted. Although in the past omitting it, was confusing later on when we tried to supersede it.

As this is now signed already that's ok. Next one will then have to be revision 2, if any changes are required. Also maybe it does make sense to have assertions being 1-indexed, rather than 0-indexed.